packages/components/organisms/f-header/src/components/_tests/Logo.test.js

```diff
@@ -44,6 +44,31 @@ describe('Logo', () => {
         expect(logo).toBeDefined();
     });
 
+    it('should change the logo link to a span if linkDisabled prop is provided', () => {
```

Comment:
Do we need the reverse of this? i.e when it is a href or is that covered?

Reply:
Done 🙂